lib/anony/anonymisable.rb

```diff
@@ -54,6 +54,23 @@ def valid_anonymisation?
         @anonymise_config.valid?
       end
 
+      # Finds the records that relate to a particular subject and runs anonymise on
+      # each of them. If a selector is not defined it will railse an exception.
+      def anonymise_for!(subject, subject_id)
+        anonymise_config.
+          select(subject, subject_id, &:anonymise!)
```

Comment:
Is it worth double checking that each `subject` responds to `anoynmise!`?

Reply:
Can do - what behaviour would you expect? An error being raised?

Reply:
Something like this https://github.com/gocardless/anony/pull/97/commits/70d18ab9bd038acedd6fbb188f69e2dbf4e640df?